### ISSUE TITLE
Should Include GCC High and DoD Required Endpoints

### ISSUE THREAD

**Issue by Freddy-Santos** [MicrosoftDocs/OfficeDocs-SharePoint#2486]
[Enter feedback here]
This page contains the specific requirements for the Migration Manager Agent when using a Government Cloud, however, the Setup page that originally referenced this link only list the required endpoints for the commercial cloud.  It would be helpful to list the correct Government Cloud endpoints on this page so that all the prerequisites are listed in one place.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7ceaa921-290f-7730-e787-2ee040d616ae
* Version Independent ID: a0140e51-138c-7cf5-1f6c-ac66a0588a6c
* Content: [Migration Manager Government Cloud settings - Migrate to SharePoint and OneDrive](https://docs.microsoft.com/en-us/sharepointmigration/mm-gov-cloud)
* Content Source: [migration/mm-gov-cloud.md](https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/blob/live/migration/mm-gov-cloud.md)
* Service: **sharepoint-online**
* GitHub Login: @JoanneHendrickson
* Microsoft Alias: **jhendr**

**Comment by get-itips** [comment]
Hello @Freddy-Santos 
A new article including the required Gov endpoints is now live here https://docs.microsoft.com/en-us/sharepointmigration/mm-prerequisites
Please let me know if that helps your inquiry.
Thanks

**Comment by JoanneHendrickson** [comment]
Hi @Freddy-Santos  I have updated the Government Cloud settings article to include the Government specific endpoints and included a link directing users to the full list of Prerequisites and required Endpoints article that @get-itips noted.  We want to make sure customers understand they need the full list. Thanks for your suggestion!

**Comment by Freddy-Santos** [comment]
Thank you @JoanneHendrickson for the update.  I did not see any other communication until I got to this link.  Perhaps the Anti-SPAM settings quarantined emails from Github without me noticing.  I click on the link above provided https://docs.microsoft.com/en-us/sharepointmigration/mm-prerequisites and noticed that those endpoints are still referencing the Commercial cloud.  I was expecting something to show the .US endpoints.  Am I looking at the wrong link?
